tethys_cli/site_commands.py

```diff
@@ -0,0 +1,197 @@
+from tethys_cli.cli_helpers import load_apps
+from django.utils import timezone
+from tethys_cli.cli_colors import write_msg
+import os
+import yaml
+from subprocess import call
+
+
+# General Settings
+TAB_TITLE = 'tab_title'
+FAVICON = 'favicon'
+BRAND_TEXT = 'title'
+BRAND_IMAGE = 'logo'
+BRAND_IMAGE_HEIGHT = 'logo_height'
+BRAND_IMAGE_WIDTH = 'logo_wigth'
+BRAND_IMAGE_PADDING = 'logo_padding'
+APPS_LIBRARY_TITLE = 'library_title'
+PRIMARY_COLOR = 'primary_color'
+SECONDARY_COLOR = 'secondary_color'
+BACKGROUND_COLOR = 'background_color'
+TEXT_COLOR = 'text_color'
+TEXT_HOVER_COLOR = 'text_hover_color'
+SECONDARY_TEXT_COLOR = 'secondary_text_color'
+SECONDARY_TEXT_HOVER_COLOR = 'secondary_text_hover_color'
+FOOTER_COPYRIGHT = 'footer_copyright'
+
+# Home Page
+HERO_TEXT = 'hero_text'
+BLURB_TEXT = 'blurb_text'
+FEATURE1_HEADING = 'feature1_heading'
+FEATURE1_BODY = 'feature1_body'
+FEATURE1_IMAGE = 'feature1_image'
+FEATURE2_HEADING = 'feature2_heading'
+FEATURE2_BODY = 'feature2_body'
+FEATURE2_IMAGE = 'feature2_image'
+FEATURE3_HEADING = 'feature3_heading'
+FEATURE3_BODY = 'feature3_body'
+FEATURE3_IMAGE = 'feature3_image'
+CALL_TO_ACTION_TEXT = 'action_text'
+CALL_TO_ACTION_BUTTON = 'action_button'
+
+arg_filter = {
+    TAB_TITLE: 'Site Title',
+    FAVICON: 'Favicon',
+    BRAND_TEXT: 'Brand Text',
+    BRAND_IMAGE: 'Brand Image',
+    BRAND_IMAGE_HEIGHT: 'Brand Image Height',
+    BRAND_IMAGE_WIDTH: 'Brand Image Width',
+    BRAND_IMAGE_PADDING: 'Brand Image Padding',
+    APPS_LIBRARY_TITLE: 'Apps Library Title',
+    PRIMARY_COLOR: 'Primary Color',
+    SECONDARY_COLOR: 'Secondary Color',
+    BACKGROUND_COLOR: 'Background Color',
+    TEXT_COLOR: 'Primary Text Color',
+    TEXT_HOVER_COLOR: 'Primary Text Hover Color',
+    SECONDARY_TEXT_COLOR: 'Secondary Text Color',
+    SECONDARY_TEXT_HOVER_COLOR: 'Secondary Text Hover Color',
+    FOOTER_COPYRIGHT: 'Footer Copyright',
+    HERO_TEXT: 'Hero Text',
+    BLURB_TEXT: 'Blurb Text',
+    FEATURE1_HEADING: 'Feature 1 Heading',
+    FEATURE1_BODY: 'Feature 1 Body',
+    FEATURE1_IMAGE: 'Feature 1 Image',
+    FEATURE2_HEADING: 'Feature 2 Heading',
+    FEATURE2_BODY: 'Feature 2 Body',
+    FEATURE2_IMAGE: 'Feature 2 Image',
+    FEATURE3_HEADING: 'Feature 3 Heading',
+    FEATURE3_BODY: 'Feature 3 Body',
+    FEATURE3_IMAGE: 'Feature 3 Image',
+    CALL_TO_ACTION_TEXT: 'Call to Action',
+    CALL_TO_ACTION_BUTTON: 'Call to Action Button'
+}
+
+
+def add_site_parser(subparsers):
+    # Setup site command
+    site_parser = subparsers.add_parser('site', help='Add/Change portal home content and display.')
```

Comment:
Add/Change portal home content and display. -> Add/Change the Tethys Portal content and theme.